src/org/labkey/test/tests/list/ListTest.java

```diff
@@ -1690,6 +1792,27 @@ private void testTricky(String listName, boolean autoKey) throws IOException
 
         validateDataRegionTableForTricky(expectedValues);
 
+        // Validates Issue 52071
+        dataRegionTable.clickRowDetails(0);
+
+        List<String> actualFields = Locator.tagWithClass("td", "lk-form-label").findElements(getDriver()).stream().map(WebElement::getText).toList();
+
+        List<String> expectedFields = new ArrayList<>();
+        // Replace the _ with a space and add a : at the end.
+        if (!autoKey)
+        {
+            expectedFields.add(keyField.replace("_", " ") + ":");
```

Comment:
Not needed since we know there's only one, but `replaceAll` seems like it would be better for future-proofing.